.github/workflows/test.yml

```diff
@@ -97,6 +96,13 @@ jobs:
         if: matrix.runner != 'windows-latest'
         run: echo | openssl s_client -showcerts -servername 'api.vercel.com' -connect 76.76.21.21:443
 
+      - name: 'Upload Test Report to Datadog'
+        if: always()
+        run: 'npx @datadog/datadog-ci@2.18.1 junit upload --service vercel-cli .junit-reports'
```

Comment:
I picked `vercel-cli` as a "service" name in Datadog because "vercel" seems too broad and I didn't have a good term for what all this repo encompass. Suggestions welcome.